Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/FeedToken/ChangeFeedIteratorCoreTests.cs

```diff
@@ -850,6 +798,291 @@ public async Task ChangeFeedIteratorCore_WithFullFidelityReadFromBeginning()
+                        Assert.AreEqual(expected: ChangeFeedOperationType.Create, actual: createOperation.Metadata.OperationType);
+                        Assert.AreNotEqual(notExpected: default, actual: createOperation.Metadata.ConflictResolutionTimestamp);
+                        Assert.AreNotEqual(notExpected: default, actual: createOperation.Metadata.CurrentLogSequenceNumber);
+                        Assert.AreEqual(expected: default, actual: createOperation.Metadata.PreviousLogSequenceNumber);
+                        Assert.IsFalse(createOperation.Metadata.TimeToLiveExpired);
+
+                        ChangeFeedItemChanges<Item> replaceOperation = itemChanges.ElementAtOrDefault(1);
+
+                        Assert.AreEqual(expected: id, actual: replaceOperation.Current.Id);
+                        Assert.AreEqual(expected: "205 16th St NW", actual: replaceOperation.Current.Line1);
+                        Assert.AreEqual(expected: "Atlanta", actual: replaceOperation.Current.City);
+                        Assert.AreEqual(expected: "GA", actual: replaceOperation.Current.State);
+                        Assert.AreEqual(expected: "30363", actual: replaceOperation.Current.ZipCode);
+                        Assert.IsNotNull(createOperation.Metadata);
+                        Assert.AreEqual(expected: ChangeFeedOperationType.Replace, actual: replaceOperation.Metadata.OperationType);
+                        Assert.AreNotEqual(notExpected: default, actual: replaceOperation.Metadata.ConflictResolutionTimestamp);
+                        Assert.AreNotEqual(notExpected: default, actual: replaceOperation.Metadata.CurrentLogSequenceNumber);
+                        Assert.AreNotEqual(notExpected: default, actual: replaceOperation.Metadata.PreviousLogSequenceNumber);
+                        Assert.IsFalse(replaceOperation.Metadata.TimeToLiveExpired);
+
+                        break;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// This test will execute <see cref="Container.GetChangeFeedIterator{T}(ChangeFeedStartFrom, ChangeFeedMode, ChangeFeedRequestOptions)"/> in <see cref="ChangeFeedMode.OperationsLog"/> (FullFidelity) with a typed item.
+        /// Using ChangeFeedStartFrom.Now(ranges[0]).
+        /// </summary>
+        [TestMethod]
+        public async Task ChangeFeedIteratorCore_FeedRange_VerifyingWireFormatTests()
+        {
+            TimeSpan ttl = TimeSpan.FromSeconds(1);
+            ContainerInternal container = await this.InitializeFFCFContainerAsync(ttl);
+            IReadOnlyList<FeedRange> ranges = await container.GetFeedRangesAsync();
+            string id = Guid.NewGuid().ToString();
+            string otherId = Guid.NewGuid().ToString();
+
+            using (FeedIterator<ChangeFeedItemChanges<Item>> feedIterator = container.GetChangeFeedIterator<ChangeFeedItemChanges<Item>>(
+                changeFeedStartFrom: ChangeFeedStartFrom.Now(ranges[0]),
```

Comment:
nit: No need to specify one of the ranges. The reason is, 1 range = 1 physical partition, you cannot predict when you call Upsert which range /partition it will go to, you might be missing the changes when limiting to 1 range.

```suggestion
                changeFeedStartFrom: ChangeFeedStartFrom.Now(),
```